ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -415,4 +415,7 @@ cat >> ${REPORT} << EOF
 <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-pink.min.css">
 <link rel="stylesheet" type="text/css" href="/static/spyglass/spyglass.css">
 EOF
-cat ${SHARED_DIR}/custom-links.txt >> ${REPORT}
+
+if [[ -f ${SHARED_DIR}/custom-links.txt ]]; then
+  cat ${SHARED_DIR}/custom-links.txt >> ${REPORT}
```

Comment:
Alternatively, is there some boilerplate content that I could insert here? `<i>"No links were collected by this CI job"</i>`?

Reply:
Yes, but if the file is present Spyglass would add a panel - and having "No links.." text there is odd

Reply:
Question is, whether it is more useful to have a user wondering why the panel is not there, or to make the panel be there and say that there are no links.